### ISSUE TITLE
Question from a vegetable chicken(muggle).......

### ISSUE THREAD

**Issue by iamastar88** [zylo117/Yet-Another-EfficientDet-Pytorch#385]
Thank you for your hard word, the code style looks neat and comfortable. But I think there may be some little problems that we haven't noticed， too many people don't train effectively. Although the network is very deep and difficult to train, there is too much instability, and the abnormal mAP of most people's training has proven it. The official code is also very deep, and although difficult to train, the results appear to be relatively normal. So maybe something should check， and maybe no problem, but it's not a bad thing to do it.

**Comment by zylo117** [comment]
Most of the troubles were caused by incorrect operations. Like wrongly convert dataset to coco style, forgetting that category id starts from 1, or just don't know how to monitor loss graph using tensorboard.

I admit the training setting is not as smart as official's.
But a model-trainer should be able to deal with this kind of issues by themselves.
I provided a basic codebase, the rest is relied on themselves.

If one reads the instructions, runs though the tutorial, follows the steps and monitor the whole training process like a pro, there won't be any big problem.

In fact, I've trained a dozens of model already with this repo. All of them perform well enough.
Maybe I will open an pinned issue to illustrate the faqs.
